### PR TITLE
[AGENT-5017] Add new "/ping" health check method compatible with SageMaker

### PR DESCRIPTION
**Pull request by oleksandr-saienko**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Added /ping method, because existing /ping/ method (with slash in the end) is not compatible with SageMaker.

### CODE REVIEW

File: custom_model_runner/datarobot_drum/drum/server.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ def base_api_blueprint(termination_hook=None):
 
     @model_api.route("/", methods=["GET"])
     @model_api.route("/ping/", methods=["GET"])
+    @model_api.route("/ping", methods=["GET"])
```
Comment (zohar-mizrahi) [inline custom_model_runner/datarobot_drum/drum/server.py:30]: Same here

**Comment by oleksandr-saienko** [comment]
Jenkins run

File: custom_model_runner/datarobot_drum/drum/runtime.py [hunk 2]
```diff
@@ -96,6 +96,7 @@ def run_error_server(host, port, exc_value):
 
     @model_api.route("/", methods=["GET"])
     @model_api.route("/ping/", methods=["GET"])
+    @model_api.route("/ping", methods=["GET"])
```
[the remark this answers is not in the document]
Comment (oleksandr-saienko) [reply custom_model_runner/datarobot_drum/drum/runtime.py]: Done

File: custom_model_runner/datarobot_drum/drum/server.py (hunk 1 above)
Comment (oleksandr-saienko) [reply custom_model_runner/datarobot_drum/drum/server.py]: Done

**Pull request merged**